elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```diff
@@ -0,0 +1,354 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: jump;
+
+$variables: (
+  // Padding: calc(#{pfe-var(container-spacer)} * 2),
+  // BackgroundColor: pfe-color(surface--base),
+);
+
+:host {
+  // Disable accordion's active borders
+  --pfe-accordion--BackgroundColor: transparent;
+  --pfe-accordion--BorderColor: #ccc;
+  --pfe-accordion--BorderColor--accent: transparent;
+  --pfe-accordion--accent: transparent;
+  --pfe-theme--color--surface--border: transparent;
+  --pfe-theme--color--ui-base--on-dark: transparent;
+  --pfe-theme--color--ui-base: transparent;
+  button{
+    text-decoration: none;
+  }
+  font-weight: #{pfe-var(font-weight--normal)};
+  display: block;
+  position: sticky;
+  top: calc(#{pfe-var(container-spacer)} * 4);
+  @media(min-width: 992px) {
+    padding: calc(#{pfe-var(container-spacer)} * 2);
+  }
+}
+
+::slotted([slot="heading"]) {
+  margin: 0;
+  margin-bottom: calc(#{pfe-var(container-spacer)} * 1);
+  font-size: #{pfe-var(font-size--heading--zeta)};
+  font-weight: #{pfe-var(font-weight--normal)};
+  text-transform: uppercase;
+  @media(max-width: 992px) {
+    display: none;
+  }
+}
+
+ul{
```

Comment:
You don't need to target the UL tag since you control that in the web component.  If you leave off the tag, then you can update the tag if you need to for semantics without updating the styles.

Reply:
Hmm... I'm not following. Are you saying use a class (or some selector that wouldn't change based on markup) instead?